integrations/quayLogin/cli/init.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash -e
+
+readonly ROOT_DIR="$(dirname "$0")/../../.."
+readonly COMMON_DIR="$ROOT_DIR/integrations/common"
+readonly HELPERS_PATH="$COMMON_DIR/_helpers.sh"
+readonly LOGGER_PATH="$COMMON_DIR/_logger.sh"
+
+# shellcheck source=integrations/common/_helpers.sh
+source "$HELPERS_PATH"
+# shellcheck source=integrations/common/_logger.sh
+source "$LOGGER_PATH"
+
+export RESOURCE_NAME=""
+export SCOPES=""
+export QUAY_USERNAME=""
+export QUAY_PASSWORD=""
+export QUAY_EMAIL=""
+
+help() {
+  echo "
+  Usage:
+    $SCRIPT_NAME <resource_name> [scopes]
+  "
+}
+
+check_params() {
+  _log_msg "Checking params"
+
+  QUAY_USERNAME="$( shipctl get_integration_resource_field "$RESOURCE_NAME" "username" )"
+  QUAY_PASSWORD="$( shipctl get_integration_resource_field "$RESOURCE_NAME" "password" )"
+  QUAY_EMAIL="$( shipctl get_integration_resource_field "$RESOURCE_NAME" "email" )"
+
+  if _is_empty "$QUAY_USERNAME"; then
+    _log_err "Missing 'username' value in $RESOURCE_NAME's integration."
+    exit 1
+  fi
+
+  if _is_empty "$QUAY_PASSWORD"; then
+    _log_err "Missing 'password' value in $RESOURCE_NAME's integration."
+    exit 1
+  fi
+
+  if _is_empty "$QUAY_EMAIL"; then
+    _log_err "Missing 'email' value in $RESOURCE_NAME's integration."
+    exit 1
+  fi
+
+  _log_success "Successfully checked params"
+}
+
+init_scope_configure() {
+  _log_msg "Initializing scope configure quay.io"
```

Comment:
I think, we use this in other integrations => `Initializing scope configure`

So could we remove `quay.io` present at the end ? ( anyway we are showing it during the `_log_grp`. So it should be clear to the user.